### PR TITLE
Added new maven plugin output content options

### LINKED ISSUE #44: gp-maven-plugin: Use only reviewed trasnlations
Add an option to include only 'reviewed' translations.

### PR DESCRIPTION
**Pull request by yumaoka**
New output content options to use only reviewed translations. Fixes #44.

### CODE REVIEW

File: gp-maven-plugin.md [hunk 1]
```diff
@@ -495,6 +495,13 @@ resource strings. This option might not be implemented by some format types. In
 When translated string value is not available, the value in the source language is used.
 * **TRANSLATED_ONLY** Emits only resource strings (with a simple header if applicable).
 When translated string value is not available, do not include the key in the output.
+* **MERGE_REVIEWED_TO_SOURCE** Duplicate the contents of the source bundle and replaces only
+translated resource strings marked as reviewed. This option might not be implemented by some
+format types. In this case, **REVIEWED_WITH_FALLBACK** is used instead.
+* **REVIEWD_WITH_FALLBACK** Emits only resource strings marked as reviewed. When transled string
```
Comment (srl295) [inline gp-maven-plugin.md]: `REVIEWD_WITH_FALLBACK` typo?

Comment (srl295) [inline gp-maven-plugin.md]: `REVIEWD_WITH_FALLBACK` typo?

File: gp-maven-plugin/src/main/java/com/ibm/g11n/pipeline/maven/GPDownloadMojo.java [hunk 2]
```diff
@@ -211,15 +211,37 @@ private void exportLanguageResource(ServiceClient client, SourceBundleFile bf, S
...
+            bundle = getBundle(client, bf.getBundleId(), language, false, false);
+            exportTranslation(bundle, language, bf.getType(), outputFile);
+            break;
+
+        case MERGE_REVIEWED_TO_SOURCE:
+            bundle = getBundle(client, bf.getBundleId(), language, true, true);
+            mergeTranslation(bundle, language, bf.getType(), bf.getFile(), outputFile);
+            break;
+
+        case REVIEWD_WITH_FALLBACK:
```
Comment (srl295) [inline gp-maven-plugin/src/main/java/com/ibm/g11n/pipeline/maven/GPDownloadMojo.java]: `REVIEWD_WITH_FALLBACK` typo?

File: gp-maven-plugin/src/main/java/com/ibm/g11n/pipeline/maven/OutputContentOption.java [hunk 3]
```diff
@@ -21,15 +21,41 @@
...
+     * is used.
+     */
+    MERGE_REVIEWED_TO_SOURCE,
+
+    /**
+     * Exports translated resources marked as reviewed. If translation
+     * is not available or, not marked as reviewed, the value from the
+     * source bundle is used.
+     */
+    REVIEWD_WITH_FALLBACK,
```
Comment (srl295) [inline gp-maven-plugin/src/main/java/com/ibm/g11n/pipeline/maven/OutputContentOption.java]: `REVIEWD_WITH_FALLBACK` typo?

File: gp-maven-plugin.md (hunk 1 above)
Comment (yumaoka) [inline gp-maven-plugin.md]: Fixed - REVIEWED_WITH_FALLBACK. Also in the same line - 'transled' -> 'translated'

File: gp-maven-plugin/src/main/java/com/ibm/g11n/pipeline/maven/GPDownloadMojo.java (hunk 2 above)
Comment (yumaoka) [inline gp-maven-plugin/src/main/java/com/ibm/g11n/pipeline/maven/GPDownloadMojo.java]: Corrected.

File: gp-maven-plugin/src/main/java/com/ibm/g11n/pipeline/maven/OutputContentOption.java (hunk 3 above)
Comment (yumaoka) [inline gp-maven-plugin/src/main/java/com/ibm/g11n/pipeline/maven/OutputContentOption.java]: Corrected.

**Pull request merged**